control_panel_api/services.py

```diff
@@ -15,35 +15,29 @@ def _policy_name(bucket_name, readwrite=False):
 
 
 def _app_role_name(app_slug):
-    """Return the IAM role name for the given `app_slug`"""
     return "{}_app_{}".format(settings.ENV, app_slug)
 
 
 def _policy_arn(bucket_name, readwrite=False):
     """Return full bucket policy arn e.g. arn:aws:iam::1337:policy/bucketname-readonly"""
-    return "{}:policy/{}".format(settings.IAM_ARN_BASE, _policy_name(bucket_name, readwrite))
+    return "{}:policy/{}".format(settings.IAM_ARN_BASE,
+                                 _policy_name(bucket_name, readwrite))
 
 
 def bucket_arn(bucket_name):
-    """Return bucket arn e.g. arn:aws:s3:::bucketname"""
     return "arn:aws:s3:::{}".format(bucket_name)
 
 
-def app_slug(name):
+def app_slugify(name):
     """Create a valid slug for s3 using standard django slugify but we add some custom"""
     return re.sub(r'_+', '-', slugify(name))
 
 
 def app_create(app_slug):
-    """Creates the IAM role for the given app"""
     _create_app_role(_app_role_name(app_slug))
 
 
 def _create_app_role(role_name):
-    """Creates the IAM role for the given app"""
-
-    # See: `sts:AssumeRole` required by kube2iam
```

Comment:
I think that comment was actually useful (it certainly saved me some research time in the lambda function version the other day!)

Reply:
Added back